highlight-range.js

```diff
@@ -84,6 +69,34 @@ function highlightRange(rangeObject, highlightTemplate) {
     return cleanupHighlights;
 }
 
+/**
```

Comment:
I’ve use JSDoc syntax here, which is not what you have elsewhere. This is a private function.